src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2414,12 +2484,9 @@
 short-summary: Place the CLI in a waiting state until a condition of a scale set is met.
 examples:
   - name: Place the CLI in a waiting state until a condition of a scale set is met. (autogenerated)
-    text: az vmss wait --created --name MyScaleSet --resource-group MyResourceGroup
+    text: |
+        az vmss wait --created --name MyScaleSet --resource-group MyResourceGroup
     crafted: true
-  - name: Place the CLI in a waiting state until the VMSS has been updated.
```

Comment:
There example shouldn't be removed.

Reply:
Yeah, it's a bug. fixing. Seems have to add them back manually for this PR. I will fix them in branch Aladdin-src

Reply:
Fixed here https://github.com/haroldrandom/azure-cli/commit/6a78f7c9491d04a75da1430066456a51aa06d8e2